src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/LambdaParameterNameCheck.java

```diff
@@ -38,6 +38,12 @@
  * <pre>
  * &lt;module name="LambdaParameterName"/&gt;
  * </pre>
+ * <p>Code Example:</p>
+ * <pre>
+ * Function&lt;String, String&gt; function1 = s -&gt; s.toLowerCase(); // OK
+ * Function&lt;String, String&gt; function2 = S -&gt; S.toLowerCase();
+ *         // violation, name 'S' must match pattern '^[a-z][a-zA-Z0-9]*$'
```

Comment:
It is better to put comment on the line where violation is, now it looks weird

Reply:
done